CollAction/Views/Projects/Details.cshtml

```diff
@@ -11,6 +12,10 @@
     var canTrack = consentFeature.CanTrack;
 
     ViewData["Title"] = @Model.Project.Name + " Details";
+
+    var user = await userManager.GetUserAsync(User);
```

Comment:
This looks like code that should be in the viewmodel, controller or service

Reply:
I moved this to the controller, and return it to the view using ViewData since it's not really a part of the project details.